spotlight-client/src/routerUtils/getUrlForResource.ts

```diff
@@ -0,0 +1,61 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2020 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+
+import assertNever from "assert-never";
+import { paramCase } from "change-case";
```

Comment:
Hyphens as word boundaries seems to be the most standard for URI paths. I did quite a bit of digging on the standard naming conventions for URIs when writing the Lantern API design doc. It seems query params are a little less standard, but the path is generally hyphens. At least thats what I found in the research that I turned up last week.

Reply:
awesome ... I found this library pretty handy for these conversions, something to keep in mind if we want to do this consistently across products (is Lantern currently following this convention or is it something you are looking to introduce?)